cypress/integration/workflows_spec.js

```diff
@@ -52,5 +55,15 @@ describe('Dashboard Workflows Page', () => {
           .to.deep.equal({Type: 'Task', Resource: '${SfSnsReportLambdaAliasOutput}', Next: 'StopStatus'});
       });
     });
+
+    it('filters workflows when a user types in the search box', () => {
+      cy.server();
+      cy.route('GET', '/workflows*').as('get-workflows');
+      cy.visit('/workflows');
+      cy.get('.table .tbody .tr').its('length').should('be.eq', 2);
+      cy.get('.table .tbody .tr').first().contains('HelloWorldWorkflow');
+      cy.get('.search').click().type('cond');
+      cy.get('.table .tbody .tr').first().contains('SecondTestWorkflow');
```

Comment:
Can we add one more line to this test checking that that length has gone down to 1?

Reply:
Sure.

Reply:
7ef28d16